src/js/utils/__tests__/StringUtil-test.js

```diff
@@ -206,6 +206,30 @@ describe("StringUtil", function() {
     });
   });
 
+  describe("#punctuate", function() {
+    it("punctuates a sentence without a period", function() {
+      expect(StringUtil.punctuate("butterfly is nice")).toEqual(
+        "butterfly is nice."
+      );
+    });
+
+    it("punctuates a single word without a period", function() {
+      expect(StringUtil.punctuate("butterfly")).toEqual("butterfly.");
+    });
+
+    it("punctuates a whitespace padded sentence without a period", function() {
+      expect(StringUtil.punctuate("butterfly is nice ")).toEqual(
+        "butterfly is nice."
+      );
+    });
+
+    it("does not further punctuate a sentence with a period", function() {
+      expect(StringUtil.punctuate("butterfly is nice.")).toEqual(
+        "butterfly is nice."
+      );
+    });
+  });
```

Comment:
I'd love to see some edge cases here. 
`StringUtil.punctuate("");`
`StringUtil.punctuate(new Array());`
`StringUtil.punctuate(null);`
`StringUtil.punctuate(1);`